docs/HACKING.md

```diff
@@ -174,86 +160,51 @@ Then, in another window
-goldstone/settings/__init__                        0      0   100%
-goldstone/settings/base                           24      3    88%
-goldstone/settings/development                     7      7     0%
-goldstone/settings/production                      1      1     0%
-goldstone/settings/stage                           1      1     0%
-goldstone/settings/test                            2      0   100%
-goldstone/urls                                     4      0   100%
-goldstone/wsgi                                     4      4     0%
-------------------------------------------------------------------
-TOTAL                                            214     50    77%
-_______________________________________ summary ___________________
-cover: commands succeeded
-congratulations :)
-```
+    OK
+    Destroying test database for alias 'default'...
+    cover runtests: commands[1] | coverage xml
+    cover runtests: commands[2] | coverage report
+    Name                                           Stmts   Miss  Cover
+    ------------------------------------------------------------------
+    goldstone/__init__                                 0      0   100%
+    goldstone/apps/__init__                            0      0   100%
+    goldstone/apps/lease/__init__                      0      0   100%
+    goldstone/apps/lease/admin                         1      0   100%
+    goldstone/apps/lease/celery                        3      3     0%
+    goldstone/apps/lease/migrations/0001_initial      18      3    83%
+    goldstone/apps/lease/migrations/__init__           0      0   100%
+    goldstone/apps/lease/models                       34      3    91%
+    goldstone/apps/lease/tasks                        21     21     0%
+    goldstone/apps/lease/tests                        77      0   100%
+    goldstone/apps/lease/tests_celery                 10      0   100%
+    goldstone/apps/lease/views                         7      4    43%
+    goldstone/libs/__init__                            0      0   100%
+    goldstone/settings                                 0      0   100%
+    goldstone/settings/__init__                        0      0   100%
+    goldstone/settings/base                           24      3    88%
+    goldstone/settings/development                     7      7     0%
+    goldstone/settings/production                      1      1     0%
+    goldstone/settings/stage                           1      1     0%
+    goldstone/settings/test                            2      0   100%
+    goldstone/urls                                     4      0   100%
+    goldstone/wsgi                                     4      4     0%
+    ------------------------------------------------------------------
+    TOTAL                                            214     50    77%
+    _______________________________________ summary ___________________
+    cover: commands succeeded
+    congratulations :)
 
 
 ### Front-end testing
```

Comment:
Two paragraphs down:

"In addition, the e2e testing suite is required to log in in order to received an auth token "

"in in"

"receive" is misspelled.